src/components/SkillBar.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { formatName } from "../utilities";
+
+export const SkillBar = ({ skillName, ranks }) => {
```

Comment:
Nitpick: It's already implied that `skillName` is going to be the name due to it being in `SkillBar`.  Perhaps just `skill`?

Reply:
Nice way to clean things up a little.